vector/src/main/java/im/vector/app/features/home/HomeActivityViewModel.kt

```diff
@@ -81,7 +81,8 @@ class HomeActivityViewModel @AssistedInject constructor(
         private val analyticsStore: AnalyticsStore,
         private val lightweightSettingsStorage: LightweightSettingsStorage,
         private val vectorPreferences: VectorPreferences,
-        private val analyticsTracker: AnalyticsTracker
+        private val analyticsTracker: AnalyticsTracker,
+        private val analyticsConfig: AnalyticsConfig,
```

Comment:
`analyticsConfig` appear to be unused, at least there is no other change form this file in GH interface.

Reply:
it's because the previous import has been replaced with an instance property of the same name `import im.vector.app.config.analyticsConfig` -> `private val analyticsConfig: AnalyticsConfig`
 

https://github.com/vector-im/element-android/pull/6517/files#diff-7153f5b6cf4c510293d53c73474e854a3a84e6003c88341472c0808377bf3462R118

Reply:
OK, sure.